src/renderer/state/test/mocks.ts

```diff
@@ -425,7 +425,7 @@ export const mockSuccessfulUploadJob: UploadJob = {
 
 export const mockFSSUploadJob: FSSUpload = {
   created: new Date(),
-  currentStage: "ADDING CHUNKS",
+  currentStage: "UPLOAD_IN_PROGRESS",
```

Comment:
This is a meaningless value, used to instantiate the job.  The only fields that have meaning are `status`, and `service_fields.file_id`.